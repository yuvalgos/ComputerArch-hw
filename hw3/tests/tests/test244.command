./dflow_calc tests/test244.dat tests/test244.in p130 d48 d173 p196 d24 d120 d14 p190 d6 d152 p56 p140 p116 d172 d131 d93 p143 d145 d84 d115 p168 d40 d110 d7 p27 d168 p91 p86 d166 p91 d22 d106 p89 p3 d130 p7 p1 p63 p56 p116 p134 d175 d73 p135 d42 p187 p49 d116 d20 p22 d118 p51 p173 p149 p147 p112 p140 d38 d101 p150 d98 p51 d110 d106 d58 d170 p65 d139 p95 p146 d26 d48 p164 d52 p100 d180 p143 p121 p178 d163 d185 d198 d153 d138 d77 d107 p146 p150 p146 p55 p139 p142 d105 d93 d41 p8 p46 d169 d59 d156 d50 d150 d101 d184