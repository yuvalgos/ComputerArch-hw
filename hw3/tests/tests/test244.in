1 27 21 16
8 13 8 3
8 21 20 7
5 24 3 30
1 18 9 30
7 5 7 13
3 12 21 16
3 2 19 3
6 26 19 27
1 13 19 7
6 25 28 12
5 25 7 21
8 21 2 12
3 25 6 21
0 20 29 13
1 19 9 14
7 6 29 5
7 7 23 4
1 1 24 14
7 9 8 17
5 4 19 31
1 11 7 23
1 30 26 17
3 25 31 16
0 1 28 30
7 1 21 26
3 31 7 8
0 31 7 24
7 4 23 8
0 19 8 10
0 11 23 21
8 7 15 20
3 22 27 21
7 25 15 30
1 31 10 14
7 9 0 30
1 17 31 31
4 14 20 15
5 7 22 2
0 20 8 18
7 27 14 6
1 0 5 21
6 10 0 27
8 14 29 31
7 16 9 31
6 29 26 27
3 8 25 25
3 18 8 24
8 25 17 19
6 16 19 30
8 17 31 0
8 22 27 16
0 25 8 21
0 8 4 25
6 16 22 24
0 9 1 28
3 14 16 2
8 29 15 26
3 26 2 11
4 22 1 9
5 27 6 16
2 23 2 6
8 29 24 18
4 25 30 21
7 24 6 16
2 10 22 24
3 31 18 14
6 20 23 21
8 28 21 6
4 20 26 20
2 16 14 3
1 25 11 26
5 28 7 12
3 12 10 15
2 20 15 21
3 10 1 6
4 26 3 5
3 29 2 1
2 5 7 10
2 31 12 21
6 23 8 31
4 12 6 18
0 14 23 13
0 8 30 26
6 11 16 11
8 12 4 10
7 22 31 1
5 2 7 12
1 30 25 0
8 4 11 29
2 16 18 21
0 30 15 15
3 2 8 3
6 28 10 5
7 17 8 14
8 28 14 2
8 20 27 0
3 12 26 26
1 23 12 8
7 7 29 9
1 3 19 1
5 7 3 12
6 26 20 24
2 3 10 8
7 11 25 29
6 31 30 22
8 3 9 13
0 11 22 8
2 6 0 31
3 22 21 2
4 0 14 8
8 23 7 21
3 20 1 30
0 7 9 21
8 26 24 4
1 3 9 26
1 2 27 6
5 12 7 0
4 8 6 15
8 27 0 12
6 23 13 17
0 31 19 15
7 31 27 2
3 24 30 11
2 15 20 4
6 18 21 16
5 19 13 13
1 1 6 14
4 1 12 11
0 7 1 15
4 31 2 21
8 9 1 7
4 30 9 19
2 14 13 11
1 31 21 11
2 3 18 29
0 23 22 18
5 27 26 17
0 23 2 6
4 9 6 19
7 28 23 31
7 11 17 25
5 10 13 17
6 16 20 31
7 4 23 8
4 3 21 18
6 10 7 28
6 28 18 18
1 5 15 3
1 21 6 31
5 17 5 3
1 1 29 15
7 20 26 0
3 7 31 7
8 23 15 31
8 4 1 28
1 13 25 15
8 27 19 24
8 27 27 20
6 16 23 14
3 2 20 1
5 29 25 24
7 7 31 24
0 0 18 25
2 13 14 20
1 26 18 8
5 6 11 3
1 12 28 16
6 10 12 18
8 19 24 12
4 9 24 10
2 21 29 1
6 19 11 3
2 26 1 18
2 20 16 9
4 11 10 21
5 5 7 22
3 30 25 9
8 25 5 15
1 2 18 17
2 7 0 8
1 8 30 18
2 3 1 6
3 4 9 23
4 1 26 16
5 12 28 10
7 15 11 31
3 5 6 28
0 3 15 28
4 26 8 30
7 18 14 29
3 4 25 24
1 30 13 10
8 20 19 2
3 21 0 31
4 1 23 28
6 9 23 9
4 26 23 11
2 26 28 7
6 20 7 26
