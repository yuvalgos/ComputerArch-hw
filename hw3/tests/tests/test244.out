Reading the opcodes latency info from tests/test244.dat ... Got latency for 9 opcodes
Reading the program file tests/test244.in ... Found 200 instructions
getProgDepth()==272
getDepDepth(130)==200
getInstDeps(48)=={36,29}
getInstDeps(173)=={151,125}
getDepDepth(196)==251
getInstDeps(24)=={-1,22}
getInstDeps(120)=={9,94}
getInstDeps(14)=={-1,9}
getDepDepth(190)==197
getInstDeps(6)=={2,-1}
getInstDeps(152)=={114,110}
getDepDepth(56)==120
getDepDepth(140)==202
getDepDepth(116)==197
getInstDeps(172)=={141,145}
getInstDeps(131)=={128,129}
getInstDeps(93)=={75,78}
getDepDepth(143)==180
getInstDeps(145)=={12,125}
getInstDeps(84)=={70,30}
getInstDeps(115)=={55,114}
getDepDepth(168)==223
getInstDeps(40)=={37,16}
getInstDeps(110)=={82,83}
getInstDeps(7)=={-1,-1}
getDepDepth(27)==19
getInstDeps(168)=={167,125}
getDepDepth(91)==0
getDepDepth(86)==156
getInstDeps(166)=={141,145}
getDepDepth(91)==0
getInstDeps(22)=={8,-1}
getInstDeps(106)=={55,9}
getDepDepth(89)==149
getDepDepth(3)==0
getInstDeps(130)=={116,12}
getDepDepth(7)==0
getDepDepth(1)==0
getDepDepth(63)==53
getDepDepth(56)==120
getDepDepth(116)==197
getDepDepth(134)==177
getInstDeps(175)=={168,171}
getInstDeps(73)=={65,-1}
getDepDepth(135)==79
getInstDeps(42)=={41,40}
getDepDepth(187)==226
getDepDepth(49)==23
getInstDeps(116)=={60,108}
getInstDeps(20)=={15,-1}
getDepDepth(22)==20
getInstDeps(118)=={108,-1}
getDepDepth(51)==80
getDepDepth(173)==180
getDepDepth(149)==197
getDepDepth(147)==78
getDepDepth(112)==72
getDepDepth(140)==202
getInstDeps(38)=={32,7}
getInstDeps(101)=={100,97}
getDepDepth(150)==180
getInstDeps(98)=={97,83}
getDepDepth(51)==80
getInstDeps(110)=={82,83}
getInstDeps(106)=={55,9}
getInstDeps(58)=={7,30}
getInstDeps(170)=={123,168}
getDepDepth(65)==139
getInstDeps(139)=={108,126}
getDepDepth(95)==175
getDepDepth(146)==221
getInstDeps(26)=={17,-1}
getInstDeps(48)=={36,29}
getDepDepth(164)==214
getInstDeps(52)=={46,12}
getDepDepth(100)==72
getInstDeps(180)=={163,118}
getDepDepth(143)==180
getDepDepth(121)==2
getDepDepth(178)==216
getInstDeps(163)=={125,71}
getInstDeps(185)=={147,168}
getInstDeps(198)=={147,180}
getInstDeps(153)=={134,129}
getInstDeps(138)=={116,108}
getInstDeps(77)=={7,25}
getInstDeps(107)=={86,83}
getDepDepth(146)==221
getDepDepth(150)==180
getDepDepth(146)==221
getDepDepth(55)==72
getDepDepth(139)==197
getDepDepth(142)==194
getInstDeps(105)=={91,86}
getInstDeps(93)=={75,78}
getInstDeps(41)=={5,12}
getDepDepth(8)==3
getDepDepth(46)==42
getInstDeps(169)=={123,167}
getInstDeps(59)=={25,55}
getInstDeps(156)=={71,124}
getInstDeps(50)=={34,41}
getInstDeps(150)=={148,145}
getInstDeps(101)=={100,97}
getInstDeps(184)=={173,159}
